test/test_jit.py

```diff
@@ -9092,6 +9092,20 @@ def test_irparser(self):
         """
         FileCheck().run(graph_str, parse_ir(graph_str))
 
+    def test_norm_aliases(self):
```

Comment:
This test is cool but either in this PR or a follow-up we should make it extensible just like the method of registering aliases in the JIT is. That way when people add a new alias they can also easily extend this test.

Reply:
Done here: https://github.com/pytorch/pytorch/pull/38746